activestorage/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Rails 8.0.0.beta1 (September 26, 2024) ##
 
+*   Expose `connects_to` configuration for models.
+
+    *Nick Pezza*
+
```

Comment:
This should go above the `8.0.0.beta1` header since we can't change things that have already been released

Reply:
Thanks, should be all fixed now